### ISSUE TITLE
Go through the projects in istio/ecosystem

### ISSUE THREAD

**Issue by craigbox** [istio/community#805]
With Istio becoming part of the CNCF, we need to go through istio/ecosystem, and for each repository:

- archive it, if now irrelevant
- confirm it's currently part of the Istio project, or 
- move it to its own org, if it doesn't wish to become, or doesn't qualify to be, part of the Istio project in the CNCF.

**Comment by howardjohn** [comment]
Is there anything similar to SIGS we can do? I don't think we want to move
~anything under the "istio" org since that has some implication of support

**Comment by craigbox** [comment]
Yes, istio-ecosystem will remain like the kubernetes-sigs repo, but we have to check the licensing of anything being brought on board, and we have to consider if Admiral wants to have its own identity.
